source/code-snippets/typescript/dot-notation.ts

```diff
@@ -19,7 +19,7 @@ interface TestNumber {
   myNumber: number;
 }
 
-const database = client.db("<your db>");
+const database = client.db("<your database>");
```

Comment:
1. Sounds good
2. Tested this and I thought it was more readable as 2 lines than 1.

Reply:
Re 2: still not sure you need the database and collections variables unless you're targeting people new to the driver (I think you indicated to me in an earlier comment this is only for Lucas).

No change needed, but just wanted to mention that there are a few different representations that are perfectly readable used throughout the docs. For example, the Node.js engineers wrote this snippet in which both the references to the database/collection and the CRUD call are chained to the MongoClient: https://docs.mongodb.com/manual/core/transactions-in-applications/#example

Reply:
Thanks, I'll definitely consider this as an option going forward. I think it is more readable as two lines here (requires horizontal scroll with smaller window), and prettier (node teams formatter of choice) resizes this to 3 lines (each method gets its own line) rather than moves one method down.

I think two lines is best here, but if node team suggests otherwise I'll take the edit.